content/practices/names-and-descriptions/names-and-descriptions-practice.html

```diff
@@ -1360,11 +1360,11 @@ <h2><span id="naming_role_guidance_heading">Accessible Name Guidance by Role</sp
               <td>
                 <a href="#tablist" class="role-reference"><code>tablist</code></a>
               </td>
-              <td>Recommended</td>
+              <td>Required **only if** a visible label is present</td>
               <td>
                 <ul>
                   <li>Helps screen reader users understand the context and purpose of the tablist.</li>
-                  <li>Use <code>aria-labelledby</code> if a visible label is present, otherwise use <code>aria-label</code>.</li>
+                  <li>Use <code>aria-labelledby</code> if a visible label is present, otherwise optionally use <code>aria-label</code>.</li>
```

Comment:
```suggestion
                  <li>Use <code>aria-labelledby</code> if a visible label is present, otherwise optionally name with <code>aria-label</code>.</li>
```